cfgov/unprocessed/js/config/error-messages-config.js

```diff
@@ -0,0 +1,19 @@
+/* ==========================================================================
+  Error Messages Config
+
+  These messages are manually mirrored on the Python side in config.py
+   ========================================================================== */
+
+'use strict';
+
+var ERROR_MESSAGES = {
+  CHECKBOX_ERRORS : {
+    required : 'Please select at least one of the %s options.'
```

Comment:
yeah, its kind of weird because some of the error messages will be constants but others will not.